docs/versioned_docs/version-7.2.x/configuration/auth.md

```diff
@@ -336,7 +336,7 @@ you may wish to configure an authorization server for each application. Otherwis
 
 The `oidc_issuer_url` is based on URL from your **Authorization Server**'s **Issuer** field in step 2, or simply https://corp.okta.com .
 The `client_id` and `client_secret` are configured in the application settings.
-Generate a unique `client_secret` to encrypt the cookie.
+Generate a unique `cookie_secret` to encrypt the cookie.
```

Comment:
As you've updated this in the versioned docs, it won't appear in any future docs, are you able to also update the future version docs? https://github.com/oauth2-proxy/oauth2-proxy/tree/master/docs/docs

Reply:
Done - What a procedure for such a small change.